cloud/deployment/deployment.go

```diff
@@ -453,6 +461,23 @@ func Update(deploymentID, label, ws, description, deploymentName string, schedul
 		deploymentUpdate.Description = currentDeployment.Description
 	}
 
+	if dagDeploy == "enable" {
```

Comment:
Instead of checking for `dagDeploy` being enabled separately in both `Create()` and `Update()` flows, would you consider making a `isDagDeployEnabled() bool {}` that does the check? Small nit for readability.

Reply:
Making this change in: https://github.com/astronomer/astro-cli/pull/766/